src/mudlet.pro

```diff
@@ -1177,6 +1179,14 @@ LUA_LCF_L3_WORKSHOP_TABLE_ORDERED__PASS.files = $${PWD}/../3rdparty/lcf/workshop
 LUA_LCF_L3_WORKSHOP_TABLE_ORDERED__PASS.path = $${LUA_LCF_L1_WORKSHOP.path}/table/ordered_pass
 LUA_LCF.depends = mudlet
 
+# Test lua files:
+LUA_TESTS.files = \
+    $${PWD}/mudlet-lua/tests/DB.lua \
```

Comment:
Could you do the same for cmake build?

Reply:
I don't know. I could try, but I don't think I'm the right guy to make those changes.
I don't really understand the build stuff, and just made it work by copying what was done with geyser in `mudlet.pro`.
I didn't see geyser mentioned in the cmake files, so I'd just be guessing at the right thing to do. Not a good development strategy.

Reply:
> ...  so I'd just be guessing at the right thing to do. Not a good development strategy.

Well it ***has*** been employed here before... :rofl: